### DISCUSSION TITLE
Recorded impression and clicked events to interstitial and rewarded ads

### DISCUSSION THREAD

**Discussion by gianlluca** [Poing-Studios/Godot-AdMob-Android-iOS#125] [Ideas]
There's a reason to have clicked and recorded impression events to Banner ads but not on Interstitial and Rewarded?

This is a good information to collect on own analytics service! I could open a PR for that, but I wanna know first if there is a reason.

Thanks

**Comment by gumaciel** [comment]
Hi @gianlluca , really this was my fault, there's no reason not to have it, and it would be an honor if you open a PR for this.

Don't forget that you also need to change it in the Plugin Editor: https://github.com/Poing-Studios/godot-admob-editor

If possible also open a PR for iOS: https://github.com/Poing-Studios/godot-admob-ios but if you can't, no problem, I can do it here
